serpentTools/parsers/depmatrix.py

```diff
@@ -126,11 +116,15 @@ def __processDenseMatrix(self, stream, matrixSize):
         return line
 
     def __processSparseMatrix(self, stream, matrixSize):
-        cscProcessor = SparseCSCStreamProcessor(stream, DEPMTX_REGEX, longfloat)
+        from scipy.sparse import csc_matrix
+        from serpentTools.parsers.sparseReader import SparseCSCStreamProcessor
+
+        cscProcessor = SparseCSCStreamProcessor(
+            stream, DEPMTX_REGEX, longfloat)
         line = cscProcessor.process()
         self.depmtx = csc_matrix(
-            (cscProcessor.data[:, 0], cscProcessor.indices, cscProcessor.indptr),
-            dtype=longfloat, shape=matrixSize)
+            (cscProcessor.data[:, 0], cscProcessor.indices,
+             cscProcessor.indptr), dtype=longfloat, shape=matrixSize)
```

Comment:
@sallustius this is the section where the sparse matrix is constructed using the contents of the new `CSCStreamProcessor` object